### DISCUSSION TITLE
Training models

### DISCUSSION THREAD

**Discussion by pbussi** [deepgram/community#340] [General help] [answered]
Hi, everyone.
I need to know if the audio/video files we upload to get the transcription back is used to improve/train the AI models Deepgram is using?
Thanks!!

**Comment by jpvajda** [comment, marked as answer]
@pbussi 👋  hello Thanks for asking. 

You can refer to our[ terms of service](
https://deepgram.com/terms) for how we train our models as it relates to Training Data.
